docs/integrations/sources/s3.md

```diff
@@ -256,7 +256,8 @@ To perform the text extraction from PDF and Docx files, the connector uses the [
 
 | Version | Date       | Pull Request                                                                                                    | Subject                                                                                                              |
 |:--------|:-----------|:----------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------|
-| 4.2.1 | 2023-11-13 | [32357](https://github.com/airbytehq/airbyte/pull/32357) | Improve spec schema |
+| 4.2.2   | 2023-11-20 | [32677](https://github.com/airbytehq/airbyte/pull/32677)                                                        | Fix discovery for zip file                                                                                           |
```

Comment:
```suggestion
| 4.2.2   | 2023-11-20 | [32677](https://github.com/airbytehq/airbyte/pull/32677)                                                        | Only read files with ".zip" extension as zipped files                                                                                       |
```

Reply:
suggestion for a clearer error message since the change is not related to discovery